### ISSUE TITLE
Cannot download latest preview version

### ISSUE THREAD

**Issue by Holsann** [fetlife/android#338]
First off, this may just be a new phone issue. Recently got the S8. Fetlife app tells me that a new preview is out. Yay! Then I click, chrome opens, the page shows for a second and then goes away. Boo.

Checked my chrome settings and they seem to be the same as my last phone (S6), but wondering if I need to change an allowance setting somewhere else?

**Comment by navyguyg** [comment]
I also have an S8. After you click on the link the download should start, you should see the little down arrow in the notification area. 
Once it's done downloading you'll have to navigate to the download folder in your phone, as there have been problems with installing from the chrome browser before. It will ask if you want to install then tell you to "Enable Third Party Installation" the phone should automatically take you there and highlight the area you need to enable. Unfortunately I have had to enable this every time I install a third party app, as going in and selecting it other wise doesn't seem to stay enabled.

The app should install with no problem after that.

**Comment by Holsann** [comment]
It's definitely not downloading. Nothing shows up in the download folder. It's like my phone is just deciding to ignore it completely, which is why I wonder if I have some safety feature on that's preventing it.

**Comment by IDiedTryin** [comment]
Same issue. Click the link in the notification bar, chrome loads, briefly see the github url and gone.
Software update 2 days ago.
Galaxy S8 android 7.0  4.4.16-11854596(kernel)
Chrome 60.0.3112.107

**Comment by IDiedTryin** [comment]
Interestingly enough it (Fetlife App) says I am using the current version when I click check for updates.  4.9.7

**Comment by kistitan** [comment]
Hi all,

Some Chrome version unfortunately does not support direct download of Apps from external sources.
This issue is reported on several forums also with other Apps.

What you can do is try to use another browser or download it explicitly using the following link:

https://github.com/fetlife/android/releases/download/v2.9.11/fetlife-2.9.11.apk

If this link also does not start the download, you can just long pres on it and select "Download Link" (text might various in different browsers)

After you downloaded the Apk you can open the file in you download folder to start the installation.

You can find further details in our
Installation and Troubleshooting Guides:

https://github.com/fetlife/android/files/1127920/FetLifeAndroidAppInstallationGuide.pdf

https://github.com/fetlife/android/files/1127921/FetLifeAndroidAppFAQandTroubleshootingGuide.pdf

Let me know if it helps

**Comment by kistitan** [comment]
@IDiedTryin 

Check for updates basically should find 2.9.11.
I will look at it if I see any reason why it did not find.

Thanks for letting us know.

**Comment by IDiedTryin** [comment]
Link provided here worked fine Thanks. Could that same link be added somewhere else for us to manually click or copy as it was in the reply?
I looked and didn't find the apk link, at leeway it wasn't obvious if it was available. 
The about page in the app seems good. Or a pop-up  after the check for updates... Or page.
Or have the notification bar icon take the user to a page with the link. 
I dunno, just some suggestions.

**Comment by IDiedTryin** [comment]
Leeway = least

**Comment by kistitan** [comment]
@IDiedTryin 

As our download link changes with version, we cannot add the next version's static download link to the current version's about page, but we can add a link to our GitHub page where you can always get the latest link and other information.
We actually can also use GitHub API to receive the actual latest version's link, so that might work.

We are going to figure out something on this.
Thank you for your idea.

**Comment by Holsann** [comment]
The link worked well with the last version.  This update actually installed from clicking the app's prompt to download. Looked like it was doing the same thing and then my notifications showed it had downloaded. So yay for that!

**Comment by malachid** [comment]
Actually, while it normally works for me - it didn't for me on this last
push notification either.  The push opened chrome but not the download.  I
think it was a problem with the push payload.

With the original app, we got around changing urls by having a DNS URL
redirect rule that we pointed to instead. We would update the DNS entry,
but the redirect url was static. You might look into that.  It can easily
be tied into your build process.
